procrastinate/app.py

```diff
@@ -51,13 +57,29 @@ def __init__(
             A :py:func:`App.task` that has a custom "name" parameter, that is not
             imported and whose module path is not in this list will
             fail to run.
+        job_store:
+            **Deprecated**: Old name of ``connector``
         """
-        self.job_store = job_store
+        # Compatibility
+        if job_store:
+            message = (
+                "Use App(connector=procrastinate.PostgresConnector(...)) "
+                "instead of App(job_store=procrastinate.PostgresJobStore())"
+            )
+            logger.warn(f"Deprecation Warning: {message}")
+            warnings.warn(DeprecationWarning(message))
+            connector = job_store
+        if not connector:
+            raise TypeError("App() missing 1 required argument: 'connector'")
```

Comment:
Do we want to raise an Exception when connector and job_store are both set in the constructor arguments?

Reply:
Do you think people could end up being confused to the point they submit those 2 parameters and are not happy with the result ?

Reply:
Not sure. Up to you really.

Reply:
I vote for keeping the code simple and avoiding having one more test to write :)